### DISCUSSION TITLE
Connection issues - disconnect / connect cycles

### DISCUSSION THREAD

**Discussion by domterf** [thomluther/ha-anker-solix#153] [General]
Good morning,
first of all, thank you very much for all of your work. It adds such great value!
I don't know whether this is the right place to address this topic.
I face disconnect - connect cycles of my Solarbank 2 E1600 Pro and the Smart meter.
Is this something related to the new update or does the Anker servers have a different problem?

**Comment by domterf** [comment]
Anker seems to be having a problem with their servers.
Sorry for bothering :-)

**Comment by tibulski** [comment]
Hello,

I had the same this morning for a short time without any config changes. But now it is working again. Thanls for your great work and support!

**Comment by thomluther** [comment]
I see this all the morning, seems to be cloud server updates for the coming new features?
![image](https://github.com/user-attachments/assets/06b3a092-b581-48aa-aa38-fd811543d734)

If you only see cloud disconnects, then the server is not reachable.
If you see also Wifi disconnects, your solarbank may have problems. It may happen at cold temperatures, that Wifi chips disconnect from time to time...Then you may have to manually press the Wifi button to switch off Wifi and turn it on again, so that it will reconnect.

**Reply by domterf** [reply]
Thanks for your swift reply and for the great work!
